features/polyExplorer/src/context/explorer-context.jsx

```diff
@@ -169,6 +169,10 @@ export const ExplorerProvider = ({ children }) => {
         return entities[ppid];
     }
 
+    function entityDataRegionByPpid(ppid) {
```

Comment:
Naming nit: DataRegions are now jurisdictions?

Reply:
I thought it got changed the other way around, but jurisdictions make more sense i guess

Reply:
My point is that the same name should be used. If they are now called data regions, then data regions all the way...